### PR TITLE
docs(components): document `gs-prevalence-over-time`

### LINKED ISSUE #197: Document `gs-prevalence-over-time`

### PR DESCRIPTION
**Pull request by fengelniederhammer**
resolves #197

### Screenshot

grafik

### PR Checklist

- [x] All necessary documentation has been adapted.
- [x] The implemented feature is covered by an appropriate test.

### CODE REVIEW

File: components/src/web-components/display/prevalence-over-time-component.stories.ts [hunk 1]
```diff
@@ -33,6 +44,15 @@ const meta: Meta<PrevalenceOverTimeProps> = {
             control: { type: 'check' },
         },
     },
+    parameters: withComponentDocs({
+        componentDocs: {
+            tag: 'gs-prevalence-over-time',
```
[the remark this answers is not in the document]
Comment (fengelniederhammer) [reply components/src/web-components/display/prevalence-over-time-component.stories.ts:49]: Actually `gs-prevalence-over-time` is correct here.

File: components/src/web-components/display/prevalence-over-time-component.tsx [hunk 2]
```diff
@@ -3,27 +3,82 @@ import { customElement, property } from 'lit/decorators.js';
...
 import { PreactLitAdapterWithGridJsStyles } from '../PreactLitAdapterWithGridJsStyles';
 
+/**
+ * This component displays the prevalence over time of one or more variants.
+ * The prevalence is calculated as the ratio of the number of cases of each variant given as `numerator`
+ * to the number of cases of the variant given as `denominator`.
+ *
+ * In the chart views,
+ * - the user can select whether to display a confidence interval (not available in the bubble chart).
+ * - the user can select the y-axis scale (linear, logistic, logit).
```
[the remark this answers is not in the document]
Comment (JonasKellerer) [reply components/src/web-components/display/prevalence-over-time-component.tsx:16]: Good idea. Can you add a ticket?

File: components/src/web-components/display/prevalence-over-time-component.stories.ts
Comment (fengelniederhammer) [inline components/src/web-components/display/prevalence-over-time-component.stories.ts:1]: Reminder: Squash before merge

File: components/src/web-components/display/prevalence-over-time-component.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (fengelniederhammer) [reply components/src/web-components/display/prevalence-over-time-component.tsx:19]: -> #220

**Review by JonasKellerer — APPROVED** [review]
LGTM

**Pull request merged**